examples/standalone/benchmarks/run_on_daint.sh

```diff
@@ -135,7 +136,7 @@ if [ ! -d $(pwd)/${sample_cache} ] ; then
 	 if [ "$version" == "$GT4PY_VERSION" ]; then
 	     echo "copying premade GT4Py caches"
              cp -r ${premade_caches}/.gt_cache_0000* .
-             find . -name m_\*.py -exec sed -i "s|\/scratch\/snx3000\/olifu\/jenkins_submit\/workspace\/fv3core-cache-setup\/backend\/$backend\/experiment\/$experiment\/slave\/daint_submit|$(pwd)|g" {} +
+             find . -name m_\*.py -exec sed -i "s|\/scratch\/snx3000\/olifu\/jenkins_submit\/workspace\/fv3core-cache-setup\/backend\/$sanitized_backend\/experiment\/$experiment\/slave\/daint_submit|$(pwd)|g" {} +
```

Comment:
obviously you don't have to for this PR, but we duplicate this logic for copying caches. we should probably move it somewhere central. I think @twicki was planning to as part of the move to GTC, so not trying to duplicate work, just want to mention it.

Reply:
yeah, I am working on something orthogonal to it, no problem

Reply:
Indeed, so should the backend sanitizer I guess